doofinder-for-woocommerce/includes/settings/class-renderers.php

```diff
@@ -325,10 +325,13 @@ private function render_html_update_on_save($option_name)
             }
             ?>
         </select>
-        <a id="force-update-on-save" href="#" class="button-secondary update-on-save-btn update-icon">
-            <span class="dashicons dashicons-update"></span>
-            <?php _e('Update now!', 'wordpress-doofinder'); ?>
-        </a>
+        <?php
+            if (Settings::is_update_on_save_enabled()) : ?>
+                <a id="force-update-on-save" href="#" class="button-secondary update-on-save-btn update-icon">
+                    <span class="dashicons dashicons-update"></span>
+                    <?php _e('Update now!', 'wordpress-doofinder'); ?>
+                </a>
+        <?php endif;?>
```

Comment:
I would write this one as:

```suggestion
        <?php endif; ?>
```